utils/networking/src/lib.rs

```diff
@@ -17,4 +17,9 @@
 
 pub mod broadcaster;
 pub mod ip_or_socket_address;
+pub mod network_address;
+pub mod resolvable_name;
+
 pub use ip_or_socket_address::*;
+pub use network_address::*;
+pub use resolvable_name::*;
```

Comment:
Same problem with the other PR. We shouldn't use pub for both modules and re-exports. This is redundant and can cause confusion.

Reply:
I've fixed this in a follow-up RP.